architecture/additional_concepts/storage.adoc

```diff
@@ -405,6 +405,43 @@ cluster.
 
 The CLI shows the name of the PVC bound to the PV.
 
+[[pv-mount-options]]
+
+=== Mount Options
+You can specify mount options while mounting a persistent volume by using the annotation `volume.beta.kubernetes.io/mount-options`.
+
+For example:
+
+====
+[source,yaml]
+----
+  apiVersion: v1
+  kind: PersistentVolume
+  metadata:
+    name: pv0001
+    annotations:
+      volume.beta.kubernetes.io/mount-options: rw,nfsvers=4,noexec <1>
+  spec:
+    capacity:
+      storage: 1Gi
+    accessModes:
+    - ReadWriteOnce
+    nfs:
+      path: /tmp
+      server: 172.17.0.2
+    persistentVolumeReclaimPolicy: Recycle
+    claimRef:
+      name: claim1
+      namespace: default
+----
+<1> Specified mount options are then used while mounting the persistent volume to the disk.
+====
+
+[NOTE]
+====
+Fiber Channel and HostPath persistent volumes do not support mount options.
+====
+
 [[persistent-volume-claims]]
```

Comment:
Should this be rather more exhaustive list of volume types that supports `mount-options` ? Something like - https://kubernetes.io/docs/concepts/storage/persistent-volumes/#mount-options